src/tracks/Pattern.cpp

```diff
@@ -1149,7 +1149,7 @@ void PatternView::paintEvent( QPaintEvent * )
 			if( len < 0 )
 			{
 				p.drawPixmap( x, y, stepoff );
-				p.setOpacity( vol / 100.0 );
+				p.setOpacity( min(vol, 100) / 100.0 );
```

Comment:
So I did some testing, and it works really nice if you revert the pixmap to the previous, half-transparent one: 
![screenshot from 2016-08-12 00 55 47](https://cloud.githubusercontent.com/assets/6282045/17607740/fd950bce-6027-11e6-91b9-77a04d63a76d.png)

Now I also did a test without the `if(vol>100)` clause and with `p.setOpacity( vol / 200.0 );`, with your opaque pixmap:
![screenshot from 2016-08-12 00 57 26](https://cloud.githubusercontent.com/assets/6282045/17607792/4d0ca9c8-6028-11e6-9f56-001f3d1f2d60.png)
which did not look as nice because of the linear scale of the opacity, now this could be fixed by using a logscale, but I think we're much better off just drawing pixmaps on top of each other.